src/Api/Resources/ContactMailResource.php

```diff
@@ -34,6 +35,12 @@ public function store():Response
         $contactData = $this->request->get('contactData',[]);
         
         $response = $this->contactMailService->sendMail($mailTemplate, $contactData);
+
+        if($response instanceof ValidationException)
```

Comment:
Ist es sinnvoll, dass in eine Middleware auszulagern um bei allen Requests zu greifen?